packages/core/src/browser/dialogs/react-dialog.tsx

```diff
@@ -39,6 +39,16 @@ export abstract class ReactDialog<T> extends AbstractDialog<T> {
         ReactDOM.render(<>{this.render()}</>, this.contentNode, () => this.onRender.dispose());
     }
 
+    protected onAfterAttach(msg: Message): void {
+        super.onAfterAttach(msg);
+        this.update();
+    }
+
+    protected onActivateRequest(msg: Message): void {
+        super.onActivateRequest(msg);
+        this.update();
```

Comment:
activate means focus someting in already attached, displayed and rendered with, it should not be necessary